backend/seed_data/test_cases.yaml

```
@@ -86,4 +86,33 @@ test_cases:
+    module_title: "Product of Array Except Self"
+    target_runtime: O_N
+    test_cases:
+      - input: "[3,2,7,4]"
+        outputs:
+          - "[56,84,24,42]"
+      - input: "[-2,-2,-2,-2]"
+        outputs:
+          - "[-8,-8,-8,-8]"
+      - input: "[2,3,0,0,4,2]"
+        outputs:
+          - "[0,0,0,0,0,0]"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)